optking/optparams.py

```diff
@@ -108,6 +110,8 @@ def __init__(self, uod):
         self.step_type = uod.get("STEP_TYPE", "RFO")
         # variation of steepest descent step size
         self.steepest_descent_type = uod.get("STEEPEST_DESCENT_TYPE", "OVERLAP")
+        # conjugate gradient default
+        self.conjugate_gradient_type = uod.get("CONJUGTE_MODE", "FLETCHER")
```

Comment:
1) Changing the name just makes more to remember.  Let's use the keyword `conjugate_gradient_type`;
2) In any event, "CONJUGTE" is spelled incorrectly, which makes me wonder if you've tested its different values.